config/crd/bases/submariner.io_submariners.yaml

```diff
@@ -314,6 +335,33 @@ spec:
                             - public_ip
                             - subnets
                             type: object
+                          latency:
+                            description: LatencySpec describes the round trip time
+                              information in nanoseconds for a packet between the
+                              gateway pods of two clusters.
+                            properties:
+                              averageRTT:
+                                format: int64
+                                type: integer
+                              lastRTT:
+                                format: int64
+                                type: integer
+                              maxRTT:
+                                format: int64
+                                type: integer
+                              minRTT:
+                                format: int64
+                                type: integer
+                              stddevRTT:
+                                format: int64
+                                type: integer
+                            required:
+                            - averageRTT
+                            - lastRTT
+                            - maxRTT
+                            - minRTT
+                            - stddevRTT
+                            type: object
```

Comment:
Do all of these match the latest changes in submariner?

Reply:
this auto-generated, should have the latest.